build.sh

```diff
@@ -264,16 +269,39 @@ if [ ! "$NO_TEST" = 'true' ]; then
     date
     time docker run -t --rm "$GAME_IMAGE" 'printenv && ls -al'
     date
-    time docker run -t --rm "$GAME_IMAGE" "$GAME_BIN -game $GAME +version +exit" | tee "$TEST_DIR/test"
+    # srcds/cs2
+    if  [ "$APPID" = 730 ]; then
+        CONTAINER_ID=$( docker run -itd --rm "$GAME_IMAGE" "$GAME_BIN -dedicated -port 27015 +map de_dust2" )
```

Comment:
i think i accidentally added `-port 27015`, shouldn't be needed.

Reply:
yes